packages/core-utils/src/__tests__/query-params.js

```diff
@@ -38,3 +39,51 @@ describe("query-params", () => {
     });
   });
 });
+
+describe("flex-reducer", () => {
+  it("should not touch a query that doesn't include flex modes", () => {
+    expect(reduceOtpFlexModes(["WALK", "TRANSIT", "BIKE"])).toMatchSnapshot();
+  });
+  it("should modify a query that includes some flex modes", () => {
+    expect(
+      reduceOtpFlexModes(["WALK", "TRANSIT", "BIKE", "FLEX_DIRECT"])
+    ).toMatchSnapshot();
```

Comment:
You probably don't need snapshots for simple object comparison. Use `expect(...).toBe(<expected_value>)`.

Reply:
Snapshots moved in 7be6cdf4055042904e0d895d22825d5df980920f